moviescript/readme.txt

```diff
@@ -54,7 +54,6 @@ Copyright (c) 2015 by Hubert & Fischer. All rights reserved.
 Source: www.google.com/fonts
 License: This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is available with a FAQ at: http://scripts.sil.org/OFL
 
-
 == Images ==
 
 Generated with AI on Midjourney, released under the CC0 license.
```

Comment:
I still see the credits here

Reply:
I removed it through the `Edit Metadata` field that CBT now displays. I should do it manually, then.